lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1375,6 +1618,34 @@ def test_team_count(self):
             else:
                 self.assertEqual(topic['team_count'], 0)
 
+    @ddt.unpack
+    @ddt.data(
+        ('student_enrolled', True),
+        ('student_on_team_1_private_set_1', True),
+        ('student_on_team_2_private_set_1', True),
+        ('student_masters', True),
+        ('staff', True)
+    )
+    def test_teamset_type(self, requesting_user, expect_see_private_teamsets):
+        """
+        As different users, request course_1's list of topics, and see what private_managed teamsets are returned
+
+        Staff should be able to see both teamsets, and anyone enrolled in a private teamset should see that and
+        only that teamset
+
+        TODO: student_enrolled and student_masters should see no private teamsets.
+        sot1ps1 and sot2ps3 should only see their teamset
+        """
+        topics = self.get_topics_list(
+            data={'course_id': self.test_course_1.id},
+            user=requesting_user
+        )
+        private_teamsets_returned = [
+            topic['name'] for topic in topics['results'] if topic['type'] == 'private_managed'
+        ]
+        expected_private_teamsets = 2 if expect_see_private_teamsets else 0
```

Comment:
similar comment to earlier about expect_see_private_teamsets. Change it to int, calling it number_of_expected_team_sets and pass in 2 or 0 so we can assert directly.